usaspending_api/references/models.py

```diff
@@ -144,13 +146,15 @@ class SubtierAgency(models.Model):
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
     subtier_code = models.CharField(max_length=4, blank=True, null=True, verbose_name="Sub-Tier Agency Code")
+    abbreviation = models.CharField(max_length=150, blank=True, null=True, verbose_name="Agency Abbreviation")
```

Comment:
Oh, if you're talking about the verbose name, they're both referencing agencies so "Agency Abbreviation" should be fine.